_episodes/03-configuration.md

```diff
@@ -3,91 +3,178 @@ title: "Configuration"
 teaching: 0
 exercises: 0
 questions:
-- "What is user configuration file and how can I use it?"
+- "What is the user configuration file and how can I use it?"
 objectives:
```

Comment:
I would probably rephrase this question to "What is the user configuration file and how do I use it?" Otherwise it sounds a little too optional for me, like it would actually be possible to ignore the user configuration file and run the ESMValTool without it.